disable components in options & component disabling from config.yaml

jankal opened the pull request:
This changes make it possible to disabled components by adding them to `disabled` in the options object whilst re-adding old functionality of disabling components from `config.yaml`.

jankal committed:
feat: Re-enable component disabling from config.yaml; Disable components through the `disabled` opti

pauloamgomes commented:
Hi @jankal , thanks for you PR, but I'm afraid it will not work because the getDisabled() function was removed in the meantime.
But I'm not closing the door to a change, can you kindly open an feature request issue and detail the reason to have the disable of components in the config?

If what you need is an exclusion mode instead of inclusion, that is supported by cockpit, you can set in your field definition something like:

```json
{
"exclude": ["component1", "component2"]
}
```

jankal commented:
@pauloamgomes I've just added a commit to re-add the `getDisabled` function back into the addon. I didn't notice, that it was also removed because we are currently running some bodged-together version of the addon where multiple things were edited.

As for the reason why the option to disable components through the config should remain:    
We do have multiple page/post types (`collections`) in which we do use layouts for the page contents. As a matter of fact, we don't use the standard components included with cockpit but build our own ones with the LayoutComponents addon by @aheinze. As of now I would have to disable the standard components for every layout field or would have to enable all custom components for every layout field again.

I think, this is not good enough, because I do want to keep a list of globally disabled components (which is esentially easier than a list of enabled plugins in my case), to do the disabling for every layout field. Whenerver @aheinze is now adding a component to cockpit, I would have to update my list only once.    
As of now, I would have to update multiple lists (one for every collection with a layout) which takes more time.

I don't think, this PR is in acceptable state right now, as I will have to remove the `disabled` part I added in. Thanks for hinting me to `exclude`. I will add this the my inofficial docs I'm in the process of writing right now.
